pom.xml

```diff
@@ -296,5 +296,72 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>openliberty</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <db.driverClass>org.hsqldb.jdbc.JDBCDriver</db.driverClass>
+        <db.jdbcUrl>jdbc:hsqldb:file:./cargo-tracker-data/cargo-tracker-database;create=true</db.jdbcUrl>
+      </properties>
+      <dependencies>
+        <!-- Provided dependencies -->
+        <dependency>
+          <groupId>org.eclipse.microprofile</groupId>
+          <artifactId>microprofile</artifactId>
+          <version>6.0</version>
+          <type>pom</type>
+          <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-api</artifactId>
+            <version>10.0.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.hsqldb</groupId>
+          <artifactId>hsqldb</artifactId>
+          <version>2.7.2</version>
+          <scope>provided</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
Why is this needed?

Reply:
it is required by [src/main/java/org/eclipse/cargotracker/application/util/RestConfiguration.java](https://github.com/eclipse-ee4j/cargotracker/blob/53f32dd96dd929b255b9264bff95ef739caf4026/src/main/java/org/eclipse/cargotracker/application/util/RestConfiguration.java#L7)